cluster-scaling/README.adoc

```diff
@@ -0,0 +1,89 @@
+= Kubernetes Cluster Scaling
+:toc:
+:imagesdir: ../images
+
+This tutorial will walk you through how to setup cluster scaling using `cluster-autoscaler` to enable worker node autoscaling based on multiple metrics within Kubernetes.
+
+== Preliminary Steps
+
+Each worker running in your Kubernetes cluster MUST have the necessary IAM Roles assigned. These differ base on how you'd like to configure the autoscaler.
+
+The first set of permissions and used to enable autoscaling of a defined ASG.
+
+  {
+    "Version": "2012-10-17",
+    "Statement": [
+      {
+        "Effect": "Allow",
+        "Action": [
+          "autoscaling:DescribeAutoScalingGroups",
+          "autoscaling:DescribeAutoScalingInstances",
+          "autoscaling:SetDesiredCapacity",
+          "autoscaling:TerminateInstanceInAutoScalingGroup"
+        ],
+        "Resource": "*"
+      }
+    ]
+  }
+
+The other configuration requires an additional permission to allow the cluster to `auto discover` which ASG based on the ASG tags. This configuration looks like this:
+
+  {
+    "Version": "2012-10-17",
+    "Statement": [
+      {
+        "Effect": "Allow",
+        "Action": [
+          "autoscaling:DescribeAutoScalingGroups",
+          "autoscaling:DescribeAutoScalingInstances",
+          "autoscaling:DescribeTags",
+          "autoscaling:SetDesiredCapacity",
+          "autoscaling:TerminateInstanceInAutoScalingGroup"
+        ],
+        "Resource": "*"
+      }
+    ]
+  }
+
+To configure these permissions you need to create the policies then associate them with the worker node instance profile.
+
+  aws iam create-policy --policy-document file://cluster-scaling/templates/asg-policy.json --policy-name ClusterAutoScaling
```

Comment:
The URL does not seem correct, missing `.`?

Reply:
URL? like the `file://path`? if you `cat cluster-scaling/templates/asg-policy.json` from the root it works?

Reply:
`cat` would, but don't we need `file://./cluster-scaling/templates/asg-policy.json`?

Reply:
At least on macOS the `aws` cli works w/o the `./` not sure linux

Reply:
Good enough then ;)